ui/hooks/src/use-close-notification.ts

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+export function useCloseNotification(key: string, initialValue = false) {
+  const [closed, setClosed] = useState(() => {
```

Comment:
I would prefer to extract this callback function outside of the hook and provide the initialValue as param